.github/workflows/02-publish-artifacts.yaml

```diff
@@ -46,53 +46,34 @@ jobs:
           CLIENT: release
           CGO_ENABLED: 1
         run: python3 --version && python3 build.py
- 
-      - name: Upload release binaries (Windows / MacOS)
+
+      - name: Upload release assets (Windows / MacOS)
         id: upload-release-asset-win-macos
-        uses: actions/upload-release-asset@v1
+        uses: shogo82148/actions-upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ inputs.upload_url }}
-          asset_path: build/${{ matrix.os }}/kubescape
-          asset_name: kubescape-${{ matrix.os }}
-          asset_content_type: application/octet-stream
+          asset_path: build/${{ matrix.os }}/*
         if: matrix.os != 'ubuntu-20.04'
 
-      - name: Upload release binaries (Linux)
+      - name: Upload release assets (Linux)
         id: upload-release-asset-linux
-        uses: actions/upload-release-asset@v1
+        uses: shogo82148/actions-upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ inputs.upload_url }}
-          asset_path: build/ubuntu-latest/kubescape
-          asset_name: kubescape-ubuntu-latest
-          asset_content_type: application/octet-stream
+          asset_path: build/ubuntu-latest/*
         if: matrix.os == 'ubuntu-20.04'
 
-      - name: Upload release hash (Windows / MacOS)
-        id: upload-release-hash-win-macos
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ inputs.upload_url }}
-          asset_path: build/${{ matrix.os }}/kubescape.sha256
-          asset_name: kubescape-${{ matrix.os }}-sha256
-          asset_content_type: application/octet-stream
-        if: matrix.os != 'ubuntu-20.04'
-
-      - name: Upload release hash (Linux)
-        id: upload-release-hash-linux
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  publish-krew-plugin:
```

Comment:
@dwertent how can I add an if to only run this one from kubescape/kubescape repo ?

Reply:
@matthyx You can add:
```
if: "${{ github.repository_owner }} == kubescape"
```

Reply:
Let's merge this PR once you complete this final  adjustment